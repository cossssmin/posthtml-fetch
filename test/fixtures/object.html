<fetch url="https://jsonplaceholder.typicode.com/users/1">
  User name: {{ response.name }}
  City (nested key): {{ response.address.city }}
</fetch>
