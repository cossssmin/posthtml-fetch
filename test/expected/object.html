User name: Leanne Graham
  City (nested key): Gwenborough
